From 8a3f47243adc6601d25bcc8aec17db325900dc68 Mon Sep 17 00:00:00 2001
From: Andy Duan <fugang.duan@nxp.com>
Date: Mon, 6 Mar 2017 13:41:54 +0800
Subject: [PATCH 1690/5242] MLK-14334 ARM: dts: move regulator reg_vsd_3v3b to
 imx7ulp-evk-sd1.dts file

commit  360ecf24442f33065c7ea1df7ad884d3954f4de1 from
https://source.codeaurora.org/external/imx/linux-imx.git

Since regulator reg_vsd_3v3b for SD1, and has pin confict with lpuart,
then move the regulator to SD1 specific dts file.

Signed-off-by: Fugang Duan <fugang.duan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp-evk-sd1.dts |   14 ++++++++++++++
 arch/arm/boot/dts/imx7ulp-evk.dts     |    9 ---------
 2 files changed, 14 insertions(+), 9 deletions(-)

diff --git a/arch/arm/boot/dts/imx7ulp-evk-sd1.dts b/arch/arm/boot/dts/imx7ulp-evk-sd1.dts
index a081809..1dddfb214 100644
--- a/arch/arm/boot/dts/imx7ulp-evk-sd1.dts
+++ b/arch/arm/boot/dts/imx7ulp-evk-sd1.dts
@@ -9,6 +9,20 @@
 
 #include "imx7ulp-evk.dts"
 
+/ {
+	regulators {
+		reg_vsd_3v3b: regulator@2 {
+			compatible = "regulator-fixed";
+			reg = <2>;
+			regulator-name = "VSD_3V3B";
+			regulator-min-microvolt = <3300000>;
+			regulator-max-microvolt = <3300000>;
+			gpio = <&gpio2 11 GPIO_ACTIVE_HIGH>;
+			enable-active-high;
+		};
+	};
+};
+
 &bcmdhd_wlan_0 {
 	status = "disabled";
 };
diff --git a/arch/arm/boot/dts/imx7ulp-evk.dts b/arch/arm/boot/dts/imx7ulp-evk.dts
index cd87a87..046c3c2 100644
--- a/arch/arm/boot/dts/imx7ulp-evk.dts
+++ b/arch/arm/boot/dts/imx7ulp-evk.dts
@@ -93,15 +93,6 @@
 			enable-active-high;
 		};
 
-		reg_vsd_3v3b: regulator@2 {
-			compatible = "regulator-fixed";
-			reg = <2>;
-			regulator-name = "VSD_3V3B";
-			regulator-min-microvolt = <3300000>;
-			regulator-max-microvolt = <3300000>;
-			gpio = <&gpio2 11 GPIO_ACTIVE_HIGH>;
-			enable-active-high;
-		};
 	};
 
 	extcon_usb1: extcon_usb1 {
-- 
1.7.9.5

